controller.go

```diff
@@ -476,12 +479,23 @@ func (c *controller) ID() string {
 
 func (c *controller) BuiltinDrivers() []string {
 	drivers := []string{}
-	for _, i := range getInitializers(c.cfg.Daemon.Experimental) {
-		if i.ntype == "remote" {
-			continue
+	c.drvRegistry.WalkDrivers(func(name string, driver driverapi.Driver, capability driverapi.Capability) bool {
+		if driver.IsBuiltIn() {
+			drivers = append(drivers, name)
 		}
-		drivers = append(drivers, i.ntype)
-	}
+		return false
+	})
+	return drivers
+}
+
+func (c *controller) BuiltinIPAMDrivers() []string {
```

Comment:
We added the `BuiltInDriver` API for docker/docker to make use of. But we missed adding it for IPAM drivers. This API is added so that docker/docker can make use of it. Infact, this is required to fix one of the plugin related issues that I will push a PR in docker/docker once this PR is merged and vendored.